app/src/main/java/tech/relaycorp/gateway/data/database/StoredParcelDao.kt

```diff
@@ -27,22 +29,28 @@ interface StoredParcelDao {
     @Query(
         """
         SELECT * FROM Parcel
-        WHERE recipientLocation = :recipientLocation 
+        WHERE recipientLocation = :recipientLocation AND expirationTimeUtc > :expiresSince
```

Comment:
How about we use `date('now')` instead of `:expiresSince`? That parameter isn't currently used by the code calling this method, and it probably shouldn't be set anyway.

According to the SQLite docs, `date('now')` uses UTC: https://www.sqlite.org/lang_datefunc.html

This comment also applies to `listForRecipients`.

Reply:
We're comparing against epoch milliseconds, so from what I could test on sqlite we would need to compare with something like `datetime(expirationTimeUtc / 1000, 'unixepoch' 'utc') > datetime('now', 'utc')` I believe.

I thought about that. But the fact that we're storing in epoch milliseconds is specified in the kotlin side, on the Converters. So I though I shouldn't be leaking that abstraction to SQL and then locking it there too.

Reply:
I see. I didn't remember SQLite doesn't have proper data types for dates! Let's leave it as it is in that case.